src/rules/function-url-quotes/__tests__/index.js

```diff
@@ -72,146 +76,7 @@ testRule(rule, {
     code: "a { background-image: url(\"foo.css\"), url(\"bar.css\"), url(\"baz.css\"); }",
   }, {
     code: "a { background-image: url( \"foo.css\" ), url( \"bar.css\" ), url( \"baz.css\" ); }",
```

Comment:
Yep, line 256 and onwards. I merged the `"double"` and `"single"` testing into `"always"`.

Is that what you mean?